vassal-app/src/main/java/VASSAL/build/widget/PieceSlot.java

```diff
@@ -83,6 +84,8 @@
 public class PieceSlot extends Widget implements MouseListener, KeyListener {
   public static final String GP_ID = "gpid"; //NON-NLS
   private static final int DEFAULT_SIZE = 64;
+  private static Image noImage;
```

Comment:
Is there a reason for not making this final and assigning it here?

Reply:
Nope. Fixed.